### PR TITLE
Visitor can see side articles

### PR DESCRIPTION
**Pull request by mantanquzzmo**
[PT-Link](https://www.pivotaltracker.com/story/show/170646666)

- Adds DisplaySideArticles component
- Adds redux

image

### CODE REVIEW

File: cypress/integration/visitorCanSeeSideArticles.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  });
+  
+  it("successfully", () => {
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:side_articles_shown.json"
+    });
+    cy.visit("/");
+    cy.get("#side-articles-div-2").should("contain", "TestBody2");
```
Comment (oliverochman) [inline cypress/integration/visitorCanSeeSideArticles.js]: add an assertion that we see 3 article elements in the sidebar div

File: cypress/integration/visitorCanSeeSideArticles.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  });
+
+  it("unsuccessfully", () => {
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:side_articles_empty.json"
+    });
+    cy.visit("/");
+    cy.get("#message2").should("contain", "No articles found");
```
Comment (oliverochman) [inline cypress/integration/visitorCanSeeSideArticles.js]: please come up with something more descriptive for that element `id`

File: src/components/DisplaySideArticles.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  if (props.sideArticles && props.sideArticles.articles.length > 0) {
+    props.changeCurrentArticleId(props.sideArticles.articles[0].id);
+  }
+
+  let articlesList;
+
+  if (props.sideArticles && props.sideArticles.articles.length > 0) {
+    let articlesList = props.sideArticles.articles.map(x => {
```
Comment (oliverochman) [inline src/components/DisplaySideArticles.jsx]: change `x` to something more descriptive

**Review by oliverochman — CHANGES_REQUESTED** [review]

**Review by oliverochman — COMMENTED** [review]
Please add a screenshot

File: src/components/DisplaySideArticles.jsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      return (
+        <div id={`side-articles-div-${x.id}`} key={x.id}>
+          <p id="article-title">{x.title}</p>
+          <p id="article-body">{x.body}</p>
```
Comment (tochman) [inline src/components/DisplaySideArticles.jsx]: x

File: cypress/integration/visitorCanSeeSideArticles.js (hunk 2 above)
Comment (mantanquzzmo) [reply cypress/integration/visitorCanSeeSideArticles.js]: changed to error-message. As this div will only contain error messages from the api.

File: src/components/DisplaySideArticles.jsx (hunk 3 above)
Comment (mantanquzzmo) [reply src/components/DisplaySideArticles.jsx]: yes

File: src/components/DisplaySideArticles.jsx (hunk 4 above)
Comment (mantanquzzmo) [reply src/components/DisplaySideArticles.jsx]: yes

File: cypress/integration/visitorCanSeeSideArticles.js (hunk 1 above)
Comment (mantanquzzmo) [reply cypress/integration/visitorCanSeeSideArticles.js]: four asserted in this PR as it corresponds to user story.

**Review by tochman — APPROVED** [review]

**Pull request merged**